packages/components/src/components/input-wth-checkbox/input-with-checkbox.jsx

```diff
@@ -43,13 +43,29 @@ const InputWithCheckbox = ({
 
     // eslint-disable-next-line consistent-return
     React.useEffect(() => {
+        const showErrorToast = () => {
```

Comment:
Nit, you could have a check for `isMobile` at the top and `return` if `false` to avoid defining these (unused) functions.